docs/en/interfaces/cli.md

```diff
@@ -65,6 +65,31 @@ You can cancel a long query by pressing Ctrl+C. However, you will still need to
 
 The command-line client allows passing external data (external temporary tables) for querying. For more information, see the section "External data for query processing".
 
+### Formatted Queries {#cli-formatted-queries}
```

Comment:
This feature is usually called prepared statements, not formatted queries. This impacts the whole following text.
Though as far as I can see there's currently no actual preparation, so maybe "Queries with parameters" like in original PR might look better.

Reply:
I've called it "Formatted" because we have a dial with formatted query string similar to format strings in Python or in C (printf). There is no any other functionality. Queries with parameters sounds strange for me because we don't talk about queries, we talk about interfaces of data I/O.

Reply:
It should be similar to other DBMS, not other programming languages. Queries are part of public interface.